README.md

```diff
@@ -37,15 +37,67 @@ To run the tests:
 docker run --rm multipy multipy/runtime/build/test_deploy
 ```
```

Comment:
can we add a comment that the built files will be in multipy/runtime/build ?

Reply:
Yeah this one is a bit weird. Some of the files are being copied out from build/dist to opt/dist while the rest of build is not.